src/analyze.ts

```diff
@@ -25,9 +25,9 @@ const asyncWalk: typeof import('../node_modules/estree-walker').asyncWalk = requ
 
 // Note: these should be deprecated over time as they ship in Acorn core
 const acorn = Parser.extend(
-  require("acorn-class-fields"),
-  require("acorn-static-class-features"),
-  require("acorn-private-class-elements")
+  //require("acorn-class-fields"),
+  //require("acorn-static-class-features"),
+  //require("acorn-private-class-elements")
```

Comment:
Should we fully remove these lines as tests are passing with them disabled?

Reply:
I'm going to do a follow up PR to make sure we support all Node.js 16 features and therefore plugins